### ISSUE TITLE
"Module not found" when using --watch flag

### ISSUE THREAD

**Issue by Lerious** [OpenNTF/dojo-webpack-plugin#269]
I'm using v2.8.12 and am lazy loading some modules. When I save changes to one of those modules  in `--watch` mode and do a hard refresh, the error "Module not found" is thrown when loading it. I'm also using Babel loader.

```
13.077da2d9698e29cebe18.js:18565 Error: Module not found: core-js/modules/es.array.index-of
```

This issue doesn't happen when doing a regular build.

**Comment by chuckdumont** [comment]
Not much to go on.  Could you share your webpack config, or else provide a reproducible test case?

**Comment by MarkH817** [comment]
Hi, I'm on the same team as @Lerious.

Here's a recreation of the issue:
- https://github.com/MarkH817/dojo-webpack-plugin-sample/tree/watch-mode-issue
- [Diff](https://github.com/MarkH817/dojo-webpack-plugin-sample/compare/master...watch-mode-issue)

---

The issue only occurs while using `--watch` for webpack. In `js/lazy.js`, I'm using `String.prototype.padStart`, which is polyfilled by Babel.

- `npm run watch`
- In the `release` directory, the bundle for that file is calling webpack's internal `__webpack_require__` to load the Babel polyfill, as it should.
- Save changes in that file.
  - The bundle rebuilds and throws an error when loaded in the browser.
- The bundle for that file _now_ calls for the polyfill using `require`, instead of `__webpack_require__`, which is the bug we found.

Hopefully that helps demonstrate our issue, and thank you for this plugin!

**Comment by chuckdumont** [comment]
Interesting.  Not sure what's going on yet, but you can work around the issue by adding the following to the dojo-webpack-plugin options: 

```javascript
cjsRequirePatterns: [/(imports-loader|exports-loader)[?!]/, /core-js/]
```

Note that the first regex just replaces the default value as mentioned in [cjsRequirePatterns](https://github.com/OpenNTF/dojo-webpack-plugin#cjsrequirepatterns).  The second regex is the one that tells dojo-webpack-plugin to treat the require call inserted by babel as CommonJS.

**Comment by chuckdumont** [comment]
Issue is fixed in 2.8.13.  You should no longer need the cjsRequirePatterns workaround.
